kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -627,85 +656,54 @@ define([
 
         /**
          * builds contents of panel-heading div with the log controls in it
-         * @param {Object} events
          */
-        renderControls(events) {
+        renderControls() {
+            const buttons = [
```

Comment:
Refactored this code to generate the buttons so that it is less repetitive. The event listeners are now added in the `start` method.